pkg/proxy/proxier.go

```diff
@@ -325,6 +325,10 @@ func NewProxier(loadBalancer LoadBalancer, listenIP net.IP, iptables iptables.In
 		return nil
 	}
 	glog.Infof("Setting Proxy IP to %v", hostIP)
+	return CreateProxier(loadBalancer, listenIP, iptables, hostIP)
+}
+
+func CreateProxier(loadBalancer LoadBalancer, listenIP net.IP, iptables iptables.Interface, hostIP net.IP) *Proxier {
```

Comment:
This should not be a public symbol - createProxier() or newProxierInternal() please